build/cloud/05-configmap.yaml

```diff
@@ -7,33 +7,22 @@ metadata:
     k8s-app: kubeedge
     kubeedge: cloudcore
 data:
-  controller.yaml: |
-    controller:
-      kube:
-        master: https://kubernetes.default.svc.cluster.local:443
-        kubeconfig: /etc/kubeedge/cloud/kubeconfig.yaml
-        namespace: ""
-        content_type: "application/vnd.kubernetes.protobuf"
-        qps: 5
-        burst: 10
-        node_update_frequency: 10
-    cloudhub:
-      address: 0.0.0.0
-      port: 10000
-      ca: /etc/kubeedge/certs/rootCA.crt
-      cert: /etc/kubeedge/certs/edge.crt
-      key: /etc/kubeedge/certs/edge.key
-      keepalive-interval: 30
-      write-timeout: 30
-      node-limit: 10
-    devicecontroller:
-       kube:
-         master: https://kubernetes.default.svc.cluster.local:443
-         namespace: ""
-         content_type: "application/vnd.kubernetes.protobuf"
-         qps: 5
-         burst: 10
-         kubeconfig: "/etc/kubeedge/cloud/kubeconfig.yaml"
-  modules.yaml: |
+  cloudcore.yaml: |
+    apiVersion: cloudcore.config.kubeedge.io/v1alpha1
+    kind: CloudCore
+    kubeAPIConfig:
+      kubeConfig: ""
+      master: ""
     modules:
-      enabled: [devicecontroller, edgecontroller, cloudhub]
+      cloudhub:
+        nodeLimit: 10
+        tlsCAFile: /etc/kubeedge/ca/rootCA.crt
```

Comment:
It seems the `/etc/kubeedge/ca` also need mounted in container.

Reply:
rootCA.crt is mounted to /etc/kubeedge/certs/rootCA.crt
fixed